modules/xmpp/JingleSessionPC.js

```diff
@@ -69,6 +69,26 @@ export default class JingleSessionPC extends JingleSession {
             options) {
         super(sid, me, peerjid, connection, mediaConstraints, iceConfig);
 
+        /**
+         * Stores result of {@link window.performance.now()} at the time when
+         * ICE enters 'checking' state.
+         * @type {number|null} null if no value has been stored yet
+         * @private
+         */
+        this._iceCheckingStartedTimestamp = null;
+
+        /**
+         * Stores result of {@link window.performance.now()} at the time when
+         * first ICE candidate is spawned by the peerconnection to mark when
+         * ICE gathering started. That's, because ICE gathering state changed
+         * events are not supported by most of the browsers, so we try something
+         * that will work everywhere. It may not be as accurate, but given that
```

Comment:
It may not be as accurate as what? What would be the most accurate solution for Chrome, for example.